README.md

```diff
@@ -1,47 +1,79 @@
 
-**\_classes** (string): CSS class name to be applied to **List**’s containing `div`. The class must be predefined in one of the Less files. Separate multiple classes with a space. Supported classes are `"align-items-vert-center"` which aligns either the step number or image centrally, on the vertical axis, with the content.
+### \_orderedList (boolean):
+If set to `true`, each item in the list will numbered. This setting will only take affect if there are no list item images defined. The default value is `false`.
 
-**\_layout** (string): This defines the horizontal position of the component in the block. Acceptable values are `full`, `left` or `right`.
+### \_animateList (boolean):
+If set to `true`, the list of items will animate when scrolled into view. The default value is `false`.
 
-**\_animateList** (boolean): If set to `true`, the list of items will animate when scrolled into view. The default value is `false`.
+### \_percentInviewVertical (number):
+Controls what percentage of the list items height needs to be in the viewport in order for the items to animate. Default value is to animate when 70% 'in view'. You only need to set this property if you want to override the default value.
 
-**\_percentInviewVertical** (number): Controls what percentage of the list items height needs to be in the viewport in order for the items to animate. Default value is to animate when 70% 'in view'. You only need to set this property if you want to override the default value.
+### \_itemHorizontalAlignment (string):
+Controls the horizontal alignment of the list items. This setting will only take affect if the `_columns` property has a value above `0`. Values available utilise the CSS property [`justify-content`](https://developer.mozilla.org/en-US/docs/Web/CSS/justify-content). The default value is `start`. It contains the following settings:
+* `start`: Aligns the list item with the natural page direction. In a left-to-right course this is left by default.
+* `center`: Aligns the list item to the center of the container.
+* `end`: Aligns the list item to the opposite side of the natural page direction. In a left-to-right course this is right by default.
 
-**\_orderedList** (boolean): If set to `true`, each item in the list will numbered. The default value is `false`.
+### \_bulletAlignment (string):
+Controls the vertical alignment of the list item image or bullet alongside the text content. If the `_columns` property has a value above `0` then this properties alignment switches from vertical to horizontal. Values available utilise the CSS property [`align-items`](https://developer.mozilla.org/en-US/docs/Web/CSS/align-items). The default value is `start`. It contains the following settings:
```

Comment:
```suggestion
Controls the vertical alignment of the list item image or bullet alongside the text content. If the `_columns` property has a value above `0` then this property's alignment switches from vertical to horizontal. Values available utilise the CSS property [`align-items`](https://developer.mozilla.org/en-US/docs/Web/CSS/align-items). The default value is `start`. It contains the following settings:
```

Reply:
I think 'properties' is fine here.

Reply:
Brad is surely correct but I don't think it's needed anyway. Suggest dropping 'this properties'.